### DISCUSSION TITLE
How to tell if I damaged my board while flashing?

### DISCUSSION THREAD

**Discussion by 3rdFactoryUser** [emporia-vue-local/esphome#267] [Q&A]
So I was able to flash my emporia vue 2 but was only able to appear on my network but never was I able to see it on my esphome instance installed on HA. I could ping the device so I knew it was connected. After flashing it several times trying to get the configuration correct at the end of the day I accidently disconnected the ground cable while the rest were still connected. I can't seem to read logs anymore. Esphome web flasher is able to connect to something. I've tried flashing it again and it completes but still nothing. No logs or any signal. Is it possible I can replace the esp32-wrover-ib board?

**Comment by flaviut** [comment]
That's some back luck! I'd first start by trying to flash the original firmware, and see if that works.

Then it is possible, [although tricky, to replace the ESP32. You will need a hot air soldering gun](https://youtu.be/d_HpkIiWC3Y?si=SZ7JTzh7-LszIS2q&t=501).

**Reply by 3rdFactoryUser** [reply]
Flashing the original firmware didn't change it. Its seems cheaper for me to just buy the new emporia vue 3 and retry on that. And it has ethernet. Not to mention to use a hot air gun with how close the other components, seems to be require a higher degree of skill than immediate ability to not further ruin it. I'll give it an attempt later on.

**Reply by flaviut** [reply]
Sounds like a good idea. I would not personally enjoy de-soldering this device even though I have all the tools and experience needed.

> And it has ethernet.

Some caveats:

1. No one has tested or traced out the RTL8201F, so you're on your own as far as configuring it. Please do report back if you have success!
2. Ethernet in the panel is a controversial subject and depends on your local laws. It would be a good idea to consult a local electrician.
3. In any case, if you do use ethernet, [the manual requires](https://www.emporiaenergy.com/wp-content/uploads/2024/03/Emporia-Vue-Gen-3-Installation-Guide-v1.20.pdf) "Cat5e, RJ45, 600V insulation" cable.
